src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrForwardFlower.cs

```diff
@@ -375,23 +369,22 @@ await repo.CommitAsync(
                     targetBranch);
             }
 
-            return true;
+            return [];
         }
 
         result = await repo.RunGitCommandAsync(["diff", "--name-only", "--diff-filter=U", "--relative"], cancellationToken);
```

Comment:
Think this part goes against what we talked about during our code style meeting. The above part feels like early returning on a happy path